testfiles/PlantLoopHeatPump_EIR_AirSource.idf

```diff
@@ -313,8 +313,8 @@
 
   Curve:Quadratic,
     EIRCurveFuncPLR,         !- Name
```

Comment:
EIR curves should be proportional to PLR

Reply:
I'm confused. Is this just improving the example file or are there accompanying code changes that change the basis? And then it would require a transition rule?

Reply:
Yes, there was a code change to accompany this example file change. IIRC, the EIR curve wasn't being used. I'll check the code where this is now accounted for.